src/python/pants/backend/python/dependency_inference/BUILD

```diff
@@ -1,6 +1,12 @@
 # Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_sources()
+python_sources(
+    overrides={
+        "import_parser.py": {
+            "dependencies": ["./scripts/import_parser.py:resources"]
```

Comment:
Oh, this is subtly neat! Cool. Maybe worth a comment? (I jumped back and forth a few times before realizing exactly what this does)

Reply:
Also, the naming of the two files being identical, is cause for confusion..